extensions/wikia/ARecoveryEngine/js/PageFair/pageFair.js

```diff
@@ -19,19 +20,27 @@ define('ext.wikia.aRecoveryEngine.recovery.pageFair', [
 	}
 
 	function isSlotRecoverable(slotName) {
-		var result = isPageFairRecoveryEnabled() && !!recoverableSlots[slotName];
+		var result = isPageFairRecoveryEnabled() && recoverableSlots.indexOf(slotName) !== -1;
```

Comment:
What do you think of renaming `isPageFairRecoveryEnabled()` to just `isEnabled()`?